pydantic/types.py

```diff
@@ -497,3 +500,23 @@ def validate(cls, v: str) -> Any:
             raise errors.JsonError()
         except TypeError:
             raise errors.JsonTypeError()
+
+
+class IPvAnyAddress(_BaseAddress):
+    @classmethod
+    def __get_validators__(cls) -> 'CallableGenerator':
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, value: Union[str, bytes, int]) -> Union[IPv4Address, IPv6Address]:
+        try:
+            return IPv4Address(value)
+        except (AddressValueError, NetmaskValueError):
```

Comment:
```suggestion
        except ValueError:
```

Or change everywhere so we're consistent.